app/views/course/object_duplications/new.json.jbuilder

```diff
@@ -0,0 +1,17 @@
+json.currentHost current_tenant.host
+
+json.targetCourses @target_courses do |course|
+  json.(course, :id, :title)
+  json.path course_path(course)
+  json.host course.instance.host
+end
+
+json.assessmentsComponent @categories do |category|
```

Comment:
Once we add in more components, we might want to refactor this out into separate jbuilder partials also.

Reply:
I think the extra level of indirection is actually decreases readability in this case since I have to look in many files to piece together what JSON data is being generated for the current page. What do you think?

Reply:
I'm ok both ways - but following [this](https://github.com/Coursemology/coursemology2/pull/2508#discussion_r140143828), maybe keeping it all together might be good. 

We'll keep it here first bah.